### PR TITLE
elf gentx & peer

### PR DESCRIPTION
**Pull request by elfelf2**

### CODE REVIEW

File: quark/gentxs/elf-gentx.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"elf","identity":"","website":"","security_contact":"","details":""},"commission":{"rate":"0.100000000000000000","max_rate":"1.000000000000000000","max_change_rate":"1.000000000000000000"},"min_self_delegation":"1","delegator_address":"neutron1rrl76e9809df84rnfqxdflw5wks2ayqa2ej5au","validator_address":"neutronvaloper1rrl76e9809df84rnfqxdflw5wks2ayqasytdmc","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"/MWj6vex2ubG1k6wRcy5NKo1WIUIxmhBVKYH4EOmOPg="},"value":{"denom":"untrn","amount":"1000000000"}}],"memo":"8e385b86567cb86cf7a6391f7714727660bdb5da@144.91.77.194:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A0L5yfESFgl8KJgs6WdhyrwfMZhiti4Fd/OqbPaAzEG6"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["z4SW8Ab5/RcwoOX8MdgCARVnilc722SCh71OGv9xcQtxzi3F+EUDrjxyPNh9qP/bqys6yMGkvLfmzhfbfTNIoQ=="]}
```
Comment (albertandrejev) [inline quark/gentxs/elf-gentx.json:1]: Please update `max_change_rate` and `max_rate` in accordance with the documentation.

**Review by albertandrejev — CHANGES_REQUESTED** [review]

**Comment by elfelf2** [comment]
elf
https://github.com/elfelf2
elfy#5077
neutronvaloper1rrl76e9809df84rnfqxdflw5wks2ayqasytdmc

**Comment by albertandrejev** [comment]
I am afraid but it seems that your forgot to apply changes to your gentx. It is still the same.

**Comment by elfelf2** [comment]
Sorry for the confusion. I changed the gentx. Could you check it again?

**Review by albertandrejev — APPROVED** [review]

**Pull request merged**